packages/dialtone-vue2/components/avatar/avatar.vue

```diff
@@ -335,15 +333,13 @@ export default {
     showImage () {
       return this.imageLoadedSuccessfully !== false && this.imageSrc;
     },
-
-    buttonAriaLabel () {
-      if (!this.clickable) return undefined;
-
-      return this.fullName || this.imageAlt || this.$attrs['aria-label'];
-    },
   },
 
   watch: {
+    clickable () {
+      this.validateProps();
+    },
```

Comment:
Does this need immediate?

Reply:
No, the component validates the props on mounted so we're ok there